Utils/Dataflow/016_task2es/README

```diff
@@ -18,3 +18,10 @@ Comes from 009 Oracle Connector.
 3. Output
 ---------
 Goes to 019 esFormat.
+
+4. Samples
+----------
+The 'output' directory contains 2 samples:
+
+sample.ndjson - 2016 data (from 091's 2016 sample, 025 was skipped)
```

Comment:
Same as in 25: maybe update the 2016th sample too?

Reply:
See comment in #201.

Reply:
Done.